client/components/homePage/BOTM.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { Container, Image, Button, Cards } from 'react-bootstrap';
+import dummyData from "./dummyData.js";
+
+const BOTM = () => {
```

Comment:
IMO the name of this function should be more descriptive. I have no idea what it stands for.

Reply:
Agree with BOTM just didn't want to write "bookOfTheMonth" each time lol.  Have fixed webpack.  No idea what in my css could be messing with everyone else's but I will take a look now...